### PR TITLE
Add anchor link for each request in the report to make it easily shareable

### PR DESCRIPTION
**Pull request by pxd3v**
Hi everyone! I just implemented an solution for this issue: https://github.com/scanapi/scanapi/issues/260. The feature is already working, you just need to add the endpoint list index to the url and it's gonna open the page in the correct endpoint, e.g. url: ``Documents/scan-api/scanapi/scanapi-report.html#endpoint_4``.
I think that we have 2 more things to implement here: 
- would be nice to numerate the endpoints in the list, that way would be easy to the user to know the endpoint index.
- would be great to have an "share" button in the endpoint view, so the user can click on it an copy the custom url to clipboard.

I noticed another thing when editing the template HTML file, maybe would be more acessible if we start using an ``<ul>`` tag for the endpoints wrapper and ``<li>`` tags for the endpoints with proper key properties instead of ``<div>'s``.
Let me know your thoughts!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added 
+- Add anchor link for each request in the report to make it easily shareable
```
Comment (camilamaia) [inline CHANGELOG.md:10]:
```suggestion
- Add anchor link for each request in the report to make it easily shareable. [#317](https://github.com/scanapi/scanapi/pull/317)
```

**Review by camilamaia — COMMENTED** [review]
Hey! Thank you very much for your PR. I tested it here for different browsers and it works great 👌 

I believe we can create new issues for the topics you pointed out. What do you think? I will create the one for the link right now.

I've just left one minor suggestion.

**Comment by pxd3v** [comment]
So glad it worked! Was a pleasure to work on this issue and will be a pleasure to work on the "link' one if possible.

I've just commited your suggestion.

**Comment by camilamaia** [comment]
@djalmaaraujo would you mind helping me review this one? 🙇‍♀️

**Review by djalmaaraujo — APPROVED** [review]

**Comment by camilamaia** [comment]
Hey, @pxd3v thank you very much for the PR!

Also, I am going to send you an invite to join the ScanAPI org on GitHub 🚀 We invite everyone that has contributed with a merged PR in any of our repos. Here you can check our [Community Continuity Guidelines](https://github.com/scanapi/contributors/blob/master/Contributing.md) so you can understand better how it works.

And once again, thank you! 🙇‍♀️

**Comment by pxd3v** [comment]
Hi @camilamaia! it was a pleasure.

I already accepted the invite, thank you so much!! :)))

**Pull request merged**